scrapli/ssh_config.py

```diff
@@ -445,13 +447,13 @@ def __init__(self, ssh_known_hosts_file: str) -> None:
 
     def _parse(self) -> Dict[str, Dict[str, str]]:
         """
```

Comment:
The original docstring here appears to have been copied from SSHConfig._parse() above and never changed, so let's bring it in line with the other comments in the SSHKnownHosts class.

Reply:
heh, copy/paste -- sounds like me 😁 good catch!